apps/teams-test-app/e2e-test-data/meeting.json

```diff
@@ -340,6 +340,29 @@
       },
       "expectedAlertValue": "updateMicState called with micState: isMicMuted:true, reason:1",
       "expectedTestAppValue": "updateMicState called with micState: isMicMuted:true"
+    },
+    {
+      "title": "joinMeeting API Call - Success",
+      "type": "callResponse",
+      "version": ">2.20.0",
+      "boxSelector": "#box_joinMeeting",
+      "inputValue": {
+        "joinWebUrl": "https://www.example.com/",
+        "source": "Other"
+      },
+      "expectedAlertValue": "joinMeeting called with joinWebUrl: \"https://www.example.com/\"",
+      "expectedTestAppValue": "joinMeeting called with joinWebUrl: \"https://www.example.com/\""
+    },
+    {
+      "title": "joinMeeting API Call - Error",
+      "type": "callResponse",
+      "version": ">2.20.0",
+      "boxSelector": "#box_joinMeeting",
+      "inputValue": {
+        "joinWebUrl": "https://www.example.com/",
+        "source": "Other"
```

Comment:
Could you please explain what's the difference between successful case and this error case? Why it should show the alert that joinWebUrl or threadId is not passed?

Reply:
This test required a few changes after removal of threadId from parameters list. Made those, now if joinWebUrl is not passed (which is a required field), it should throw an error.